app/services/hex/comparer.rb

```diff
@@ -0,0 +1,50 @@
+module Hex
+  class Comparer
+    RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
+    ACCENT_MODIFIERS = [1.14, 1.08, 1.06, 0.96, 0.9, 0.8, 0.7, 0.6].freeze
+
+    def initialize(hexes)
+      @hexes = hexes.sort
+    end
+
+    def smallest
+      hexes.first
+    end
+
+    def biggest
+      hexes.last
+    end
+
+    def brightness(amount = 1)
+      rgb = hex_to_rgb_hash(smallest).transform_values do |color|
+        (color * amount).round
+      end
+      format("#%<r>02x%<g>02x%<b>02x", rgb)
+    rescue StandardError
+      smallest
+    end
+
+    # Returns the first valid hex string it finds (# + 6 digits)
+    def accent(amount = 1)
+      ACCENT_MODIFIERS.each do |modifier|
+        with_brightness = brightness(modifier**amount)
+        break with_brightness if with_brightness.size == 7
+      end
+    end
```

Comment:
I added the `amount` argument here since I removed it from `initialize`. We don't currently modify this value in the code, but it is referenced in this method. I figured it didn't hurt to leave the flexibility in place since it's already here, even if we aren't using it right now.